frontend/src/components/Connections/Connections.js

```diff
@@ -0,0 +1,61 @@
+
+
+/* eslint-disable */
+import React, { Component } from 'react';
+
+import Image from './Connection/image';
+import OptionModal from './Connection/imageModal';
```

Comment:
I think it'd be more convenient if we named the component and the file name same thing  (ImageModal) make sense?